examples/src/main/java/com/algorand/algosdk/example/Main.java

```diff
@@ -19,15 +19,29 @@
 import com.algorand.algosdk.transaction.Transaction;
 import com.algorand.algosdk.util.Encoder;
 
-import java.security.Security;
+import com.squareup.okhttp.HttpUrl;
+
 import java.math.BigInteger;
 
 
+
 public class Main {
 
     public static void main(String args[]) throws Exception {
-        final String ALGOD_API_ADDR = "http://localhost:8080";
-        final String ALGOD_API_TOKEN = "b3ff7b3765bf33c7557a2cd334238eca07578e3ce266cb03fad93d12aab0622e";
+        if (args.length != 2) {
+            System.err.println("Required parameters: ALGOD_API_ADDR ALGOD_API_TOKEN");
+            System.err.println("The parameter values can be found in algod.net and algod.token files within the data directory of your algod install.");
+            System.exit(1);
+        }
```

Comment:
put examples for parameters to give people an idea what to put to the command line

Reply:
I was hoping that the README file was fulfilling this request. 
The maven command in the README file gives an argument template: 
127.0.0.1:8080 X-Algo-API-Token
Putting random characters as an example for the token may not be very helpful.

Reply:
maybe change README to ${X_Algo_API_Token} then?